pkg/keystore/file/key.go

```diff
@@ -83,6 +83,7 @@ func encryptKey(k *ecdsa.PrivateKey, password string, edg keystore.EDG) ([]byte,
 		}
 		addr = a
 	case elliptic.P256():
+		//lint:ignore SA1019 to be addressed soon
```

Comment:
Actually, I believe that the most correct is to set the ethereum address to the empty string if the curve is not btcec.S256(), as that makes sense only for that curve. The switch was added on #3797 and elliptic.Marshal does not provide a valid ethereum address.